package.json

```
@@ -64,5 +64,23 @@
   },
   "dependencies": {
     "semver": "^7.3.2"
+  },
+  "contributes": {
+    "configuration": {
+      "type": "object",
+      "title": "npm Outdated",
+      "properties": {
+        "npm-outdated.version": {
```

`version` doesn't feel right to me, what about something like range or level?

```suggestion
        "npm-outdated.range": {
```

---

`level` might sound appropriate since range usually denotes value from min to max